Tasks/Xcode/task.loc.json

```diff
@@ -11,8 +11,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 4,
-    "Minor": 127,
```

Comment:
Is it just me, or are you disturbed that the task.json and the task.loc.json were out of sync?

Reply:
Hmm I didn't notice that and it doesn't show up on my local either! That is incorrect, I will check.